VERSION

```diff
@@ -0,0 +1 @@
+1.4.0
```

Comment:
This just feels weird to have two places to have the version's it should just get the meta-resin version from one place and an append from here.